.github/workflows/CD.yml

```diff
@@ -75,7 +75,8 @@ jobs:
         run: |
           pip install --upgrade pip
           pip install numpy==1.26.4
-          pip install geopandas shapely networkx owslib map2model loopprojectfile beartype gdal==3.8.4 hjson pytest scikit-learn
+          pip install geopandas shapely networkx owslib map2model beartype gdal==3.8.4 pytest scikit-learn
```

Comment:
I think we may not need to pin gdal anymore, would be good to test if 3.8.4 is required or if we could just use >3.8